.github/workflows/copy-santised-productionBackup.yml

```diff
@@ -0,0 +1,89 @@
+name: Restore Sanitised Production Backup to Preprodcution
+
+on:
+  schedule: # 03:00 UTC
+    - cron: "0 3 * * *"
+
+jobs:
+  backup:
+    name: Backup PaaS Database ( Production )
+    runs-on: ubuntu-latest
+    services:
```

Comment:
The ticket asks for the existing database backup workflow to be used (`.github/workflows/database-backup.yml`).  We could move this services block along with the `Sanitise the Database backup` and `Upload Sanitised Backup` steps into that workflow so that we don't have duplicate code.  This would also avoid running 2 backups at the same time.

Reply:
yes, make sense